data/core.yaml

```diff
@@ -1831,10 +1831,13 @@ en:
         reference: Intersecting highways should share a junction vertex.
     ambiguous_crossing_tags:
       title: Ambiguous Crossing Tags
-      message: "{feature} crossing info conflicts with {feature2}"
-      incomplete_message: "{feature} has crossing info, but one or more of its nodes does not."
-      tip: "Crossing way markings conflict with crossing nodes"
-      reference: Crossing way markings should not conflict with its crossing node.
+      message: "{feature} line info conflicts with point {feature2}"
```

Comment:
Reviewers: I'm particularly interested in ways that I can make the error text clearer. This is a little difficult since we're using template strings that change depending on what the entities in question are. Sometimes `{feature}` will be 'Marked Crossing (Zebra)`, for example.

Reply:
Yeah this is tricky.. I'd probably try to avoid inserting the feature name like "Marked Crossing (Zebra)" .  

I normally don't like asking the user to make a specific decision about an OSM tag, but this is probably one of those situations where we can't avoid it.  Simpler is better..

Maybe like 
```
 Line has tag "zebra" but point has tag "unmarked" 
 🔧  Set both to "zebra"
 🔧  Set both to "unmarked"
 ❌  Ignore this issue
```

Reply:
All righty! I've updated the wording for all three checks to be much more terse and to-the-point. 
Ambiguous crossings now get: 
![image](https://github.com/facebook/Rapid/assets/1887955/a7097535-9417-4445-a769-f5bb4121ef5e)

Fix text now refers to points/lines, and clearly states that the fix is to change the point into a crossing point. 

For the direct conflicts, the text is much simpler and avoids mentioning the crossing entity titles: 
![image](https://github.com/facebook/Rapid/assets/1887955/b864d41a-977d-41f7-be0c-762b7dae0d54)

Marked vs. Unmarked conflicts now get: 

![image](https://github.com/facebook/Rapid/assets/1887955/6ed5d638-b46c-4c4a-a92b-4dc090baca52)

Reply:
I'll leave this convo unresolved to ask about this new wording, @bhousel. It's better, but I'm not sure everything's perfect yet.